resources/views/bread/partials/form-builder.blade.php

```diff
@@ -0,0 +1,116 @@
+@php
+    $dataTypeRows = $dataType->{(isset($dataTypeContent->id) ? 'editRows' : 'addRows' )};
+    $formBuilderFields = collect([]);;
+@endphp
+<!-- form start -->
+<form role="form"
+        class="form-edit-add"
+        action="@if(isset($dataTypeContent->id)){{ route('voyager.'.$dataType->slug.'.update', $dataTypeContent->id) }}@else{{ route('voyager.'.$dataType->slug.'.store') }}@endif"
+        method="POST" enctype="multipart/form-data">
+    <!-- PUT Method if we are editing -->
+    @if(isset($dataTypeContent->id))
+        {{ method_field("PUT") }}
+    @endif
+
+    <!-- CSRF TOKEN -->
+    {{ csrf_field() }}
+
+    @foreach($formBuilder as $key => $htmlRow)
+        <div class="{{ $htmlRow->class }}">
+            @foreach($htmlRow->panels as $key_panel => $panel)
+                <div class="{{ $panel->class ?? '' }}">
```

Comment:
Great job :) 

I think this is only available in PHP 7 and up, If you change it to use the old ternary operator I'm sure all the tests will pass and we can merge this in. https://lornajane.net/posts/2015/new-in-php-7-null-coalesce-operator

Thanks.